src/Mosaic/Mosaic.tsx

```diff
@@ -5,13 +5,14 @@ import { GridContainer, MosaicItem } from './Mosaic.style'
 
 export const Mosaic = React.forwardRef<HTMLDivElement, MosaicProps>(
   ({ children, spacing, rounded, columns, ...rest }, ref) => {
-    const createMosaic = (items: React.ReactNode[]) => (
+    const CreateMosaic = (items: React.ReactNode[]) => (
       <MosaicItem
         data-items-length={items.length}
         data-spacing={spacing}
         data-rounded={rounded}
+        key={React.useId()}
```

Comment:
[The documentation](https://reactjs.org/docs/hooks-reference.html#useid) explicitly advise against this usage of `useId`.

Reply:
My bad, I change it!